nuget/AppCenter.nuspec

```diff
@@ -31,7 +31,7 @@
         <dependency id="System.Configuration.ConfigurationManager" version="4.6.0"/>
       </group>
       <!-- Use empty dependency groups for iOS and Android so that additional dependencies aren't added to them -->
-      <group targetFramework="MonoAndroid403">
+      <group targetFramework="MonoAndroid50">
```

Comment:
Not sure about this change. Did you test nugets with it? Is it version of Android or MonoAndroid?

Reply:
Yes, I checked it. All works well. I guess the `monoAndroid50` should match `Android 5`.